test/test_jit_cuda_fuser.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+from contextlib import contextmanager
+
+import torch
+
+from torch.testing._internal.common_utils import run_tests, ProfilingMode, GRAPH_EXECUTOR, skipIfRocm
+
+from test_jit import JitTestCase, RUN_CUDA
+
+
+if GRAPH_EXECUTOR == ProfilingMode.PROFILING:
+    torch._C._jit_set_profiling_executor(True)
+    torch._C._jit_set_profiling_mode(True)
+
+@contextmanager
+def enable_new_fuser():
+    old_cpu_fuse = torch._C._jit_override_can_fuse_on_cpu(False)
```

Comment:
`_jit_override_can_fuse_on_cpu` is bound to `overrideCanFuseOnCPU`, which returns `void`, so this line won't do what you expected it to do. I recently exposed `_jit_can_fuse_on_cpu`(a bind of `canFuseOnCPU`, and similar for GPU) for using in such cases.